bin/test.bash

```diff
@@ -73,7 +73,7 @@ cleanup() {
     echo "Builder terminated."
   fi
   # Exit the script
-  exit 130
+  exit $EXIT_CODE
```

Comment:
There is still an issue here... the `$EXIT_CODE` is still going to be undefined if `cleanup` is called from a `Ctrl-C`.  You aren't seeing the same error as earlier because `$EXIT_CODE` is not quoted (`"$EXIT_CODE"`).  I don't so much care about the quotes here as _string splitting_ shouldn't be an issue.  But the exit code should be `130` in this case.  So maybe just add a conditional here that if `$EXIT_CODE` is not defined then set it to `130`.

Reply:
> There is still an issue here... the `$EXIT_CODE` is still going to be undefined if `cleanup` is called from a `Ctrl-C`. You aren't seeing the same error as earlier because `$EXIT_CODE` is not quoted (`"$EXIT_CODE"`). I don't so much care about the quotes here as _string splitting_ shouldn't be an issue. But the exit code should be `130` in this case. So maybe just add a conditional here that if `$EXIT_CODE` is not defined then set it to `130`.

Yeah, I'll get to adding a conditional before putting it up for review, I don't know why I added the "" to the exit code, I honestly think I was mentally autopiloting but I am curious if there's a way to pass in default parameters like in Python (though Bash is for scripting, less for programming)

Reply:
Added Ad-Hoc conditional.